inference-engine/src/gna_plugin/backend/gna_limitations.hpp

```diff
@@ -31,10 +31,16 @@ constexpr uint32_t maxPoolMaxWindowSize = 6;
 constexpr uint32_t copyMaxGrouping = 8;
 constexpr uint32_t transposeMaxSize = 65528;
 
+inline bool IsTranspose2d(const std::vector<size_t>& shape) {
+    auto shape_no_1 = shape;
+    shape_no_1.erase(std::remove(shape_no_1.begin(), shape_no_1.end(), 1), shape_no_1.end());
```

Comment:
maybe count or count_if instead of copying and erasing vector twice?

Reply:
done